parsec/api/protocol/user.py

```diff
@@ -19,31 +19,28 @@
     "device_claim_serializer",
     "device_cancel_invitation_serializer",
     "device_create_serializer",
-    "device_revoke_serializer",
 )
 
 
 #### Access user API ####
 
 
-class UserGetReqSchema(BaseReqSchema):
-    user_id = UserIDField(required=True)
+class TrustchainSchema(UnknownCheckedSchema):
```

Comment:
So the trustchain doesn't use the `SCHEMA_CLS` API?

Reply:
No, the `Trustchain` class is just a helper internal to the backend (like `UserInvitation` for instance)